src/messaging/classes/Numbers.ts

```diff
@@ -38,20 +38,18 @@ export class Numbers extends HttpClient {
             const validate = new Validator<TSubscriptionCreateRequest>(
                 subscription
             );
-            validate.schemaRef('ProvisionNumberRequest').schemaInline({
+            // validate.schemaRef('ProvisionNumberRequest');
```

Comment:
for hygiene, can we remove this commented line now?

Reply:
thx, I have removed the comment